network-api/networkapi/buyersguide/templates/fragments/product-criterion.html

```diff
@@ -0,0 +1,20 @@
+<div class="criterion criterion-{{ value|lower }} criterion-{% if value == bad %}bad{% else %}good{% endif %}">
```

Comment:
This seems like you're just outputting `criterion-bad criterion-bad` or `criterion-good criterion-good`. The existing template outputs the criterion name?

Reply:
Ah, I think I see now. This has to do with the bad attribute being passed in to the fragment?